projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_base.py

```diff
@@ -45,6 +45,8 @@ def __init__(
         op_id: str,
         ingester: IIngesterPlugin,
         ingest_config: IngesterConfiguration,
+        pre_processors: List[Optional[IIngesterPlugin]] = [],
```

Comment:
default initialization seems redundant, and also IDE displays messages

Reply:
Fixed, thanks for pointing it out